sourcegraph/sourcegraph.proto

```diff
@@ -142,7 +146,9 @@ message Repo {
 	// Fork is whether this repository is a fork.
 	bool fork = 12;
 
-	// Mirror is whether this repository is a mirror.
+	// Mirror indicates whether this repo's canonical location is on
+	// another server. Mirror repos track their upstream and are not
+	// eligible for discovery on this server.
 	bool mirror = 13;
```

Comment:
Personally I don't understand the difference but it might be simply because I'm not familiar with the greater scheme of how things work. Up to you, if you feel you can add more documentation, I think that is *always* welcome. Otherwise all of this LGTM